modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -666,7 +666,19 @@
 				@export var car_label = "Speedy"
 				@export var car_number = 3
 				[/codeblock]
-				[b]Note:[/b] Subgroups cannot be nested, they only provide one extra level of depth. Just like the next group ends the previous group, so do the subsequent subgroups.
+				[b]Note:[/b] Subgroups themselves cannot be nested, but you can use the [code]/[/code] separator to achieve the desired effect.
```

Comment:
Nitpick, feel free to contest.
```suggestion
				[b]Note:[/b] Subgroups cannot be nested, but you can use the slash separator ([code]/[/code]) to achieve the desired effect:
```
I was trying to be a bit more explicit, but I kind of don't like it:
```suggestion
				[b]Note:[/b] This annotation cannot be used to nest subgroups, but you can use the slash separator ([code]/[/code]) to nest subgroups:
```

Reply:
The first suggestion looks good, thanks. But let me disagree on the second one. "This annotation cannot be used to nest subgroups" is completely wrong, because the example below shows how you can use `@export_subgroup` to create nested subgroups. It just doesn't look like what the user would expect, i.e. the following won't work:

```gdscript
@export_subgroup("Subgroup")
@export_subgroup("Nested Subgroup")
@export_subgroup("") # End "Nested Subgroup"
@export_subgroup("") # End "Subgroup"
```

Perhaps my original wording is not clear enough, but at least it's not false. I was trying to emphasize that `@export_subgroup` is not a start/end marker for a subgroup, but rather a switch where you have to specify the full path each time (and you can go back to the level of any ancestor). If you can suggest an alternative wording that is more clear but still concise, I would greatly appreciate it.

I also think this is a good reason to clarify in the documentation that the `prefix` is ​​trimmed in the inspector, allowing you to reduce visual noise in the editor.